lib/fork_break/process.rb

```diff
@@ -4,16 +4,19 @@ class << self
       attr_accessor :breakpoint_setter
     end
 
+    attr_reader :return_value
+
     def initialize(debug = false, &block)
       @debug = debug
       @fork = Fork.new(:return, :to_fork, :from_fork) do |child_fork|
         self.class.breakpoint_setter = breakpoints = BreakpointSetter.new(child_fork, debug)
 
         breakpoints << :forkbreak_start
-        block.call(breakpoints)
+        returned_value = block.call(breakpoints)
         breakpoints << :forkbreak_end
```

Comment:
what would happen if returned_value was an ActiveRecord object and the parent process tried to read it?

Seems to be that one should call .to_s, else errors can happen...?